docs/our_work/ct-alignment.md

```diff
@@ -16,7 +16,7 @@ A proof-of-concept demonstrator written in Python (user interface, classical com
 
 ## Case Study
 
-This is a backup of the case study published [here](https://transform.england.nhs.uk/ai-lab/explore-all-resources/develop-ai/using-ai-to-identify-tissue-growth-from-ct-scans/) on the NHS England Transformation Directorate website.
+This is a backup of the case study published [here](https://webarchive.nationalarchives.gov.uk/ukgwa/20241101055214/https://transform.england.nhs.uk/ai-lab/explore-all-resources/develop-ai/using-ai-to-identify-tissue-growth-from-ct-scans/) on the NHS England Transformation Directorate website.
```

Comment:
For this and the others- this _is_ listed on the repo but with a link to web archive- just clarifying why this is